tests/scripts/all.sh

```diff
@@ -185,6 +185,10 @@ pre_initialize_variables () {
     # Include more verbose output for failing tests run by CMake or make
     export CTEST_OUTPUT_ON_FAILURE=1
 
+    # Save CFLAGS
+    : ${CFLAGS=}
+    export GLOBAL_CFLAGS="${CFLAGS}"
```

Comment:
This change looks highly suspicious. Why are we saving the value of `CFLAGS`? It would be very risk-prone for the behavior of `all.sh` to depend on this environment variable. If we want to have a way to change the default `CFLAGS`, it should be only a command line option.

Reply:
This change created for the build of the cloned ecp module. Use the `CFLAGS` seemed reasonable to enable the macro to build the ecp_new.c. I change it to use a command line option.